src/command_common.ts

```
@@ -0,0 +1,106 @@
+    state.info.value = state.value;
+  }
+  state.startTime = Date.now();
+  state.info.selectors = state.selectors;
+  state.info.log = state.log ? state.log : "";
+  state.info.operation = state.operation;
+  state.error = null;
+  state.screenshotId = null;
+  state.screenshotPath = null;
+  if (state.locate === true) {
+    let timeout = null;
+    if (state.options && state.options.timeout) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)